test/persist/mzcompose.py

```diff
@@ -49,16 +49,18 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 
     if args.consensus == "mem":
         consensus_uri = "mem://consensus"
-    elif args.consensus == "postgres":
-        consensus_uri = "postgresql://postgres:postgres@postgres:5432?options=--search_path=consensus"
-        c.start_and_wait_for_tcp(services=["postgres"])
-        c.wait_for_postgres(service="postgres")
+    elif args.consensus == "cockroach":
+        consensus_uri = (
+            "postgres://root@cockroach:26257?options=--search_path=consensus"
+        )
+        c.start_and_wait_for_tcp(services=["cockroach"])
+        c.wait_for_cockroach()
 
         c.sql(
             sql="CREATE SCHEMA IF NOT EXISTS consensus;",
-            service="postgres",
-            user="postgres",
-            password="postgres",
+            service="cockroach",
+            user="root",
+            password=None,
```

Comment:
This whole bit should be droppable if you specify `Cockroach(setup_materialize=True)`.

Reply:
Done, thanks.